paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -26,11 +26,11 @@ using mkldnn::memory;
 using mkldnn::primitive;
 using mkldnn::reorder;
 using mkldnn::stream;
-using platform::to_void_cast;
 using platform::GetMKLDNNFormat;
+using platform::to_void_cast;
 
 inline void GetWeightsTz(std::vector<int64_t>& weights_tz,  // NOLINT
-                         int groups, bool is_conv3d) {
```

Comment:
Hi, Do you consider rewriting this function using std::rotate? It should not be slower right?

Reply:
I will give it a try, good idea.